paystell-frontend/src/components/dashboard/analytics/StellarAnalytics.tsx

```
@@ -0,0 +1,258 @@
+
+  // --- Chart Data Preparation Logic ---
+   const chartData = useMemo(() => {
+    const formatTime = (date: Date): string => {
+      switch (timeFilter) {
+        case 'daily':
+          return format(date, 'HH:00'); // Group by hour for daily view
+        case 'weekly':
+           return format(date, 'EEE dd'); // Day of week for weekly view
+        case 'monthly':
+           return format(date, 'MMM dd'); // Day of month for monthly view
+         case 'all':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)